components/eventing-controller/pkg/backend/utils/eventmesh_types.go

```diff
@@ -1,4 +1,4 @@
-package v1alpha2
+package utils
```

Comment:
The contents of this file is an exact copy of https://github.com/kyma-project/kyma/blob/main/components/eventing-controller/api/v1alpha1/subscription_types.go#L14-L53. Is there a reason for the duplication?

Reply:
The structs defined here are related to EventMesh, so I moved them from `api` to `eventMesh_types`. 

Since we also need them in definition of subscription `v1alpha1`, they are also defined there because we cannot import the utils pkg in `v1alpha1` (circular dependency). 

So the idea is to keep the old code in `v1alpha1` and when we remove the code for `v1alpha1` later, the duplication will be removed.

Reply:
Let's add a todo above the duplicate structs to remove then when possible.

Reply:
added

Reply:
Please add the todos here as well `components/eventing-controller/pkg/backend/utils/eventmesh_types.go`.